### ISSUE TITLE
api revision version?

### ISSUE THREAD

**Issue by tuxb0x** [coronalabs/CoronaSDK-AtomPlugin#5]
To which version of the API is the autocomplete sync to.
Some functions seems to have changed and some don't even exist.
Is there a way to sync to the last corona api revision? (current one is 2017.3184)

**Comment by TJSchmitz** [comment]
I don't think it syncs at all. All the completions are stored in data/corona-data.json which, as far as I can see, isn't updated. So the only chance would be generating a new corona-data.json from the corona api-reference.
